NCSDK-16909: Add sampling period control to sensor manager (#8830)
nrfconnect/sdk-nrf: doc/nrf/libraries/caf/sensor_manager.rst

   Do not include the configuration file in other source files.
.. |device_pm_note| replace:: Not all device power states might be supported by the sensor's device.
   Check the sensor's driver implementation before configuring :c:member:`sm_sensor_config.suspend`.

> Changing sensors sample period
> ==============================

Reviewer: ```suggestion
Changing sensor sample period
=============================
```

> In order to change sensors sample period user has to send :c:struct:`set_sensor_period_event` with new periods value in ms.

Reviewer 2: In order to change sensor sample period you have to send :c:struct:`set_sensor_period_event` with new period value in milliseconds.

Reviewer: ```suggestion
To change the sensor sample period, you must send :c:struct:set_sensor_period_event with a new period value in milliseconds.
```

> To identify which sensors sampling period you want to change pass sensors description in :c:struct:`set_sensor_period_event`.

Reviewer 2: sensors description

Reviewer: ```suggestion
To identify which sensor sampling period you want to change, set the sensor description in :c:struct:`set_sensor_period_event`.
```

> Example code changing accelerometer sampling to 400 ms.

Reviewer 2: The following code shows an example of changing accelerometer sampling to 400 ms:

   Do not include the configuration file in other source files.
.. |device_pm_note| replace:: Not all device power states might be supported by the sensor's device.
   Check the sensor's driver implementation before configuring :c:member:`sm_sensor_config.suspend`.

.. _sensor_sample_period:

Changing sensor sample period
=============================

> In order to change sensor sample period you have to send :c:struct:set_sensor_period_event with new period value in milliseconds.

Reviewer: ```suggestion
In order to change sensor sample period you have to send :c:struct:`set_sensor_period_event` with new period value in milliseconds.
```
